src/common/dsp/DPWOscillator.cpp

```diff
@@ -115,7 +115,7 @@ void DPWOscillator::process_block(float pitch, float drift, bool stereo, bool FM
              * So rather than lagging, back compute with the current dPhase
              * which makes us way more stable under phase changes
              */
-            double sBuff[3], sOffBuff[3], triBuff[3];
+            double sBuff[3]{}, sOffBuff[3]{}, triBuff[3]{};
```

Comment:
we are actually fine with this memory uninitialized since it is initialized below. Curious why you added initializers? They do add extra instructions in the inner loop here. Can you remove?